target/linux/realtek/files-5.10/drivers/net/dsa/rtl83xx/rtl838x.h

```diff
@@ -556,6 +556,13 @@ typedef enum {
 #define RTL930X_L3_IP_ROUTE_CTRL		0xab44
 
 /* Port LED Control */
+#define RTL839X_LED_SET0_0_CTRL			(0x00ec)
+#define RTL839X_LED_COPR_SET_SEL_CTRL(p)	(0x00f0 + (((p >> 5) << 2)))
+#define RTL839X_LED_FIB_SET_SEL_CTRL(p)		(0x0100 + (((p >> 5) << 2)))
```

Comment:
So `BASE + 4 * (port / 32)`. I think these bit shifts are rather difficult to read.

Reply:
That is largely a matter of taste. I really like shifting bits. But I will fix it as per your request in the update.